includes/admin/class-admin.php

```diff
@@ -1075,6 +1075,21 @@ function handle_actions() {
 						break;
 					}
 
+					$all_notices      = wpbdp_get_option( 'expiration-notices' );
```

Comment:
Can we move this new code into a separate function to avoid adding extra complexity?

Reply:
Good catch! Done, thanks!